osu.Desktop/DiscordRichPresence.cs

```diff
@@ -94,6 +96,18 @@ private void updateStatus()
             {
                 presence.State = truncate(activity.Value.Status);
                 presence.Details = truncate(getDetails(activity.Value));
+
+                if (getOnlineID(activity.Value) != string.Empty)
+                {
+                    presence.Buttons = new Button[]
+                    {
+                        new Button() { Label = "Beatmap Link", Url = $"https://osu.ppy.sh/b/{getOnlineID(activity.Value)}" }
```

Comment:
When lazer starts handling the `osu://` protocol, this should be changed to `osu://b/` and skip opening a web page. Can confirm this works in discord (just right now it opens stable's osu!direct instead of lazer's beatmap listing)

Reply:
Also, should probably go with `Open Beatmap` or similar instead:
```suggestion
                        new Button() { Label = "Open Beatmap", Url = $"https://osu.ppy.sh/b/{getOnlineID(activity.Value)}" }
```